src/utils/getPropTypes.js

```diff
@@ -0,0 +1,11 @@
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  id: PropTypes.string,
+  selector: PropTypes.string,
+  // Remove styles and classes from node.
+  shouldReset: PropTypes.bool,
+  view: PropTypes.func
+};
+
+export default () => propTypes;
```

Comment:
Any reason why this can't just export the object?

Reply:
My personal preference, but not really 😂 Will change that.

Reply:
As a note, exporting functions over constants makes things easier to change later, that's why I often use this pattern.

Reply:
@ngryman definitely, I'm in agreement here! Just thought that for proptypes it might not have been necessary. Your call in the end, though, you can keep it like this if you prefer, no strong preferences from my side :)